### ISSUE TITLE
Support for local music files

### ISSUE THREAD

**Issue by Eslyp639102** [rasmuslos/AmpFin#84]
I recently started using Ampfin and love the interface of the app and how it works. I tend to have a large majority of my music on my phone rather than on Jellyfin. I was wondering if it is possible to make AmpFin work with music files already found on my device? I would just need to import them into the app and the .lrc(lyrics) files as well. If this could be achieved, I know a lot of people that would love this app because of its sleek design. I know there are other apps that can do what I’m asking but none of them have the sleek  design and clean interface and lyrics that your app has.

**Comment by gnattu** [comment]
Isn't this what the official Apple Music app designed for?

**Comment by Eslyp639102** [comment]
> Isn't this what the official Apple Music app designed for?

No, it’s harder to get your own music files on Apple Music. Apple Music also doesn’t let you have lyrics. Plus, you need a computer to transfer all your files to your phone, but I already have all the songs on my phone. I want an app that works with the files I already have on my phone.

**Comment by rasmuslos** [comment]
AmpFin is designed to work with Jellyfin, implementing the necessary metadata providers, etc. would be possible but a big headache. That's what Jellyfin is for and I don't want to duplicate everything.
